proto/regen/ecocredit/v1/query.proto

```diff
@@ -22,46 +22,76 @@ service Query {
-    option (google.api.http).get = "/regen/ecocredit/v1/classes/{class_id}";
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/class/{class_id}"
+      additional_bindings : [ {get : "/regen/ecocredit/v1/classes/{class_id}"} ]
+    };
   }
 
   // ClassIssuers queries for the addresses of the issuers for a credit class.
   rpc ClassIssuers(QueryClassIssuersRequest)
       returns (QueryClassIssuersResponse) {
-    option (google.api.http).get =
-        "/regen/ecocredit/v1/classes/{class_id}/issuers";
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/class-issuers/{class_id}"
+      additional_bindings : [
+        {get : "/regen/ecocredit/v1/classes/{class_id}/issuers"}
+      ]
+    };
   }
 
-  // Projects queries for all projects within a class with pagination.
-  rpc Projects(QueryProjectsRequest) returns (QueryProjectsResponse) {
-    option (google.api.http).get =
-        "/regen/ecocredit/v1/classes/{class_id}/projects";
+  // ProjectsByClass queries for all projects within a class with pagination.
+  rpc ProjectsByClass(QueryProjectsByClassRequest)
+      returns (QueryProjectsByClassResponse) {
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/projects-by-class/{class_id}"
+      additional_bindings : [
+        {get : "/regen/ecocredit/v1/classes/{class_id}/projects"}
+      ]
+    };
   }
 
   // ProjectsByReferenceId queries for all projects by reference-id with
   // pagination.
   rpc ProjectsByReferenceId(QueryProjectsByReferenceIdRequest)
       returns (QueryProjectsByReferenceIdResponse) {
-    option (google.api.http).get =
-        "/regen/ecocredit/v1/projects/by-reference-id/{reference_id}";
-  }
-
-  // Project queries for information on a project.
-  rpc Project(QueryProjectRequest) returns (QueryProjectResponse) {
-    option (google.api.http).get = "/regen/ecocredit/v1/projects/{project_id}";
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/projects-by-reference-id/{reference_id}"
+      additional_bindings : [
+        {get : "/regen/ecocredit/v1/projects/reference-id/{reference_id}"}
+      ]
+    };
   }
 
   // ProjectsByAdmin queries for all projects by admin with
   // pagination.
   rpc ProjectsByAdmin(QueryProjectsByAdminRequest)
       returns (QueryProjectsByAdminResponse) {
-    option (google.api.http).get =
-        "/regen/ecocredit/v1/projects/by-admin/{admin}";
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/projects-by-admin/{admin}"
+      additional_bindings : [
+        {get : "/regen/ecocredit/v1/projects/admin/{admin}"}
+      ]
+    };
+  }
+
+  // Project queries for information on a project.
+  rpc Project(QueryProjectRequest) returns (QueryProjectResponse) {
+    option (google.api.http) = {
+      get : "/regen/ecocredit/v1/project/{project_id}"
+      additional_bindings : [
+        {get : "/regen/ecocredit/v1/projects/{project_id}"}
+      ]
+    };
   }
```

Comment:
same as above, RESTful conventions suggests plural over singular nouns for collections

Reply:
Same as above. The first get reflects the name of the query (also the CLI command) and the second is more conventional. I was thinking that maybe we would support both.